public/src/containers/create_campaign.jsx

```diff
@@ -0,0 +1,44 @@
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { reduxForm } from 'redux-form';
+
+import CampaignPage from '../components/campaign/campaign';
+
+import { saveNewCampaign } from '../actions/campaign';
+import { fetchAllScripts, fetchAllContactLists } from '../actions/campaign_form';
+
+// TODO: write validation that is applicable to campaign creation!
+// // e.g. check input length, checking uniqueness, etc
+
+function validate(values) {
```

Comment:
No, that was a relic from old code, I was trying not to change more than I had to in order to fulfill the ticket.